libdrgn/arch_x86_64.c

```diff
@@ -18,6 +18,8 @@
 
 #include "arch_x86_64_defs.inc"
 
+#define START_KERNEL_MAP 0xffffffff80000000ULL
```

Comment:
Use `UINT64_C` instead of `ULL`:
```suggestion
#define START_KERNEL_MAP UINT64_C(0xffffffff80000000)
```
Please also add a comment that this is `__START_KERNEL_map` in the kernel.

P.S. I'm amazed that this has never changed! I tracked it down to this commit in `history.git`: https://git.kernel.org/pub/scm/linux/kernel/git/history/history.git/commit/?id=0457d99a336be658cea1a5bdb689de5adb3b382d.

Reply:
I also am amazed! It's one of those things I feel dirty hardcoding, but with a solid history like that, I guess I feel a little bit better :)